.github/workflows/nightly_snapshot.yaml

```diff
@@ -37,6 +37,8 @@ jobs:
       program: "lsp"
   docs:
     uses: ./.github/workflows/build_docs.yaml
+    with:
+      rustdoc_extra_args: "-p i-slint-core -p i-slint-backend-android-activity -p i-slint-backend-winit"
```

Comment:
Since we use the same job also for the release, wouldn't this also generate i-slint-core, etc. for releases?